.github/workflows/lint.yaml

```diff
@@ -1,6 +1,14 @@
 name: Lint
 
-on: [push]
+on:
+  push:
+    branches:
+    - master
+    - develop
+  pull_request:
+    branches:
+    - master
+    - develop
```

Comment:
Fix CI double trigger.

ref: https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2

Reply:
Awesome!